test/integration/targets/setup_flatpak_remote/tasks/main.yaml

```diff
@@ -0,0 +1,12 @@
+- name: Set up dummy flatpak repository remote
+  block:
+
+    - name: Copy repo into place
+      copy:
+        src: repo
+        dest: /tmp/flatpak/
```

Comment:
Add `setup_remote_tmp_dir` to the role dependencies and use the temp dir it creates.

Reply:
:+1:

Reply:
I incorporated relying on the `setup_remote_tmp_dir` target. However, some the repository files (`.flatpakref` and `.flatpakrepo`) need to contain absolute references to the URL of the repository, which in this case is a `file:///`-reference to the repo directory on the filesystem.

The simplest solution to this is to know the target path to the repo in the file system at build time of the repository. I solved this by creating a symlink to the repo in the dynamic temp directory at test setup time at `/tmp/flatpak`, which is where the repo points.

Does this contradict what you were trying to achieve with the the dynamic temp folder? Generally speaking, these tests **are** destructive to the system under test and will not leave a clean system.

Reply:
@mattclay Are my changes covering your request?